### ISSUE TITLE
how to train an arrow datasheet with direction information

### ISSUE THREAD

**Issue by t00350320** [facebookresearch/detectron2#5482]
## 🚀 Feature
hi, I'm marking and training an arrow datasheet. In detail, label 0  means up, label 1 means right, label 2 means down, label 3 means left. 

After training, I find some prediction results are in opposite. So I guess some original arrow datas may have be changed and transformed in image augmentation stage. But not sure.

So , how to train this arrow datasheet under this case. Modify detectron2 train parameter？ Or some other measures?

training code
```
import detectron2
from detectron2.utils.logger import setup_logger
setup_logger()


from detectron2 import model_zoo
from detectron2.engine import DefaultTrainer, DefaultPredictor
from detectron2.config import get_cfg
from detectron2.data.datasets import register_coco_instances
from detectron2.evaluation import COCOEvaluator, inference_on_dataset
from detectron2.data import build_detection_test_loader


register_coco_instances(
    "my_dataset_train",  
    {},
    "dataset/annotations/instances_train2017.json",
    "dataset/train2017"  
)
register_coco_instances(
    "my_dataset_val",   
    {},
    "dataset/annotations/instances_val2017.json", 
    "dataset/val2017"  
)


cfg = get_cfg()

cfg.merge_from_file(model_zoo.get_config_file("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"))

cfg.DATASETS.TRAIN = ("my_dataset_train",)
cfg.DATASETS.TEST = ("my_dataset_val",)

cfg.OUTPUT_DIR = "./output"

cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml")

cfg.SOLVER.IMS_PER_BATCH = 2 
cfg.SOLVER.BASE_LR = 0.00025 
cfg.SOLVER.MAX_ITER = 30000  
cfg.SOLVER.STEPS = []      
cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 128 
cfg.MODEL.ROI_HEADS.NUM_CLASSES = 10 


class CocoTrainer(DefaultTrainer):
    @classmethod
    def build_evaluator(cls, cfg, dataset_name, output_folder=None):
        if output_folder is None:
            output_folder = cfg.OUTPUT_DIR
        return COCOEvaluator(dataset_name, cfg, False, output_folder)


trainer = CocoTrainer(cfg)
trainer.resume_or_load(resume=False)  
trainer.train()

```
